content/docs/getting_started/uninstall.md

```diff
@@ -130,4 +130,19 @@ Repeat the steps above for each mesh installed in the cluster. After there are n
 
 ## Removal of OSM Cluster Wide resources
 
-Uninstalling OSM will remove all the CRDs mentioned [here](https://github.com/openservicemesh/osm/tree/main/charts/osm/crds). This is the same behavior when using the Helm charts to uninstall OSM as well because OSM has a pre-delete helm hook that removes the CRDs. If there are other service meshes running in the same cluster, please ensure that you back the Custom Resources and re-apply the required CRDs and Custom Resources to the cluster using `kubectl` post a successful uninstallation of OSM.
+OSM ensures that all the CRDs mentioned [here](https://github.com/openservicemesh/osm/tree/main/cmd/osm-bootstrap/crds) exist in the cluster at install time. If they are not already installed, the `osm-bootstrap` pod will install them before the rest of the control plane components are running. This is the same behavior when using the Helm charts to install OSM as well. Uninstalling OSM will only remove/unpatch the conversion webhook from all the CRDs (which OSM adds to support multiple CR versions) and will not deleted them for primarily two reasons: CRDs are cluster-wide resources and may be used by other service meshes running in the same cluster and deletion of a CRD will cause all custom resources based off that CRD to also be deleted. 
+
+If there are no other service meshes running in the same cluster and the required custom resources have been backed up, the CRDs can be removed from the cluster using `kubectl`.
+
+Run the following `kubectl` commands:
+
+```console
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/config_meshconfig.yaml
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/config_multicluster_service.yaml
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/policy_egress.yaml
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/policy_ingress_backend.yaml
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/smi_http_route_group.yaml
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/smi_tcp_route.yaml
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/smi_traffic_access.yaml
+kubectl delete -f https://raw.githubusercontent.com/openservicemesh/osm/blob/main/cmd/osm-bootstrap/crds/smi_traffic_split.yaml
```

Comment:
Please change these to the shorter version, e.g. `kubectl delete crd tcproutes.specs.smi-spec.io`